### PR TITLE
Translated accessibility statement to French

### PR DESCRIPTION
**Pull request by nico3333fr**
- translation of page accessibility statement in french
- added a missing "." in source page, and fixed a broken link

Makes progress on #539

### CODE REVIEW

File: src/templates/fr/2019/accessibility_statement.html [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+{% block index %}
+      <ul>
+        <li><a href="#overview">Aperçu</a></li>
+        <li><a href="#measures">Measures to support accessibility</a></li>
+        <li><a href="#conformance-status">Statut de conformité</a></li>
+        <li><a href="#external-content">Contenu externe</a></li>
+        <li><a href="#compatibility">Compatibilité avec les navigateurs et les technologies d’assistance</a></li>
+        <li><a href="#technical-specifications">Spécifications techniques</a></li>
+        <li><a href="#assessment-approach">Approche d’évaluation</a></li>
+        <li><a href="#feedback">Retour d’information</a></li>
```
Comment (bazzadp) [inline src/templates/fr/2019/accessibility_statement.html:21]: Do you think we should change the anchor links? They are auto generated for the chapters so do change when you change the heading for them, but manually set for these non-chapter templates. If changing then will need to also change the ids they link to further down.

File: src/templates/fr/2019/accessibility_statement.html [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+          Le Web Almanac PDF suit les mêmes normes d’accessibilité que le site Web et, en outre, est structurellement étiqueté pour faciliter l’accessibilité.
+        </p>
+      </section>
+      
+      <section>
+        <h2 id="assessment-approach"><a href="#assessment-approach" class="anchor-link">Approche d’évaluation</a></h2>
+        
+        <p>
+          Auto-évaluation&nbsp;: le contenu a été évalué par l’équipe Web Almanac en utilisant sa propre expertise et un certain nombre d’outils dont <a href="https://wave.webaim.org/" hreflang="en">l’outil d’accessibilité WAVE</a>, le <a href="https://hacks.mozilla.org/2019/10/auditing-for-accessibility-problems-with-firefox-developer-tools/" hreflang="en">l’inspecteur d’accessibilité Firefox</a> et le <a href="https://chrome.google.com/webstore/detail/axe-web-accessibility-tes/lhdoppojpmngadmnindnejefpokejbdd" rel="nofollow">plugin AX Chrome</a>. Le Web Almanac PDF a été testé avec une analyse d’accessibilité dans Acrobat Reader Pro DC.
+      </section>
```
Comment (bazzadp) [inline src/templates/fr/2019/accessibility_statement.html:113]: Missed a `</p>` - bless that linter!:

```suggestion
        </p>
      </section>
```

File: src/templates/fr/2019/accessibility_statement.html (hunk 1 above)
Comment (nico3333fr) [reply src/templates/fr/2019/accessibility_statement.html:21]: I hesitated, I didn't know if there was something blocking on it (I was thinking to the lang switcher if there is an anchor, but it seems not). 

If there is no blocker, yes, I'd prefer to have them in french too. Will add them asap

Comment (bazzadp) [reply src/templates/fr/2019/accessibility_statement.html:21]: No blocker, so go for it!

Comment (nico3333fr) [reply src/templates/fr/2019/accessibility_statement.html]: Ok, updated link in https://github.com/HTTPArchive/almanac.httparchive.org/pull/1196/commits/cbd0158f2ce0a737ddd7b1d02b463a257ef1ee23

Comment (bazzadp) [reply src/templates/fr/2019/accessibility_statement.html]: Thanks! Fixed in https://github.com/HTTPArchive/almanac.httparchive.org/pull/1196/commits/d4fb45ccbc6e1f265c7676d347f8fd91f15ea5dc

**Comment by bazzadp** [comment]
All good from my side now. If one of the @HTTPArchive/french-translators could review the language that would be good!

File: src/templates/fr/2019/accessibility_statement.html [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+      </ul>
+{% endblock %}
+
+{% block main_content %}
+      <h1 class="title title-lg">Déclaration d’accessibilité</h1>
+      <section>
+        <h2 id="apercu"><a href="#apercu" class="anchor-link">Aperçu</a></h2>
+
+        <p>
+          Le Web Almanac s’engage à assurer l’accessibilité numérique pour tous. Nous améliorons continuellement l’expérience utilisateur pour tous et appliquons les normes d’accessibilité pertinentes.
```
Comment (borisschapira) [inline src/templates/fr/2019/accessibility_statement.html:31]: Inclusivité 

```suggestion
          Le Web Almanac s’engage à assurer l’accessibilité numérique pour tous et toutes. Nous améliorons continuellement l’expérience des personnes et appliquons les normes d’accessibilité pertinentes.
```

File: src/templates/fr/2019/accessibility_statement.html [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+        </p>
+
+        <p>
+          Nous nous considérons partiellement conformes parce que nous n’avons pas effectué un audit complet et vérifié de l’extérieur du site Web, ni que le site Web n’a été examiné par les utilisateurs de lecteurs d’écran, nous ne pouvons donc pas être sûrs de notre état actuel. Nous avons audité toutes les pages du site Web à l’aide d’un certain nombre d’outils, notamment <a href="https://wave.webaim.org/" hreflang="en">l’outil d’accessibilité WAVE</a>, <a href="https://hacks.mozilla.org/2019/10/auditing-for-accessibility-problems-with-firefox-developer-tools/" hreflang="en">l’inspecteur d’accessibilité Firefox</a> et le <a href="https://chrome.google.com/webstore/detail/axe-web-accessibility-tes/lhdoppojpmngadmnindnejefpokejbdd" rel="nofollow">plugin AX Chrome</a>.
```
Comment (borisschapira) [inline src/templates/fr/2019/accessibility_statement.html:61]: Inclusivité

```suggestion
          Nous nous considérons partiellement conformes parce que nous n’avons pas effectué un audit complet et vérifié de l’extérieur du site Web. Le site Web n’a pas non plus été examiné par des personnes utilisatrices de lecteurs d’écran, nous ne pouvons donc pas être sûrs de notre état actuel. Nous avons audité toutes les pages du site Web à l’aide d’un certain nombre d’outils, notamment <a href="https://wave.webaim.org/" hreflang="en">l’outil d’accessibilité WAVE</a>, <a href="https://hacks.mozilla.org/2019/10/auditing-for-accessibility-problems-with-firefox-developer-tools/" hreflang="en">l’inspecteur d’accessibilité Firefox</a> et le <a href="https://chrome.google.com/webstore/detail/axe-web-accessibility-tes/lhdoppojpmngadmnindnejefpokejbdd" rel="nofollow">plugin AX Chrome</a>.
```

File: src/templates/fr/2019/accessibility_statement.html [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+        </p>
+
+        <p>
+          Le but de l’Almanach Web est de fournir des informations détaillées sur l’état du Web. Bien que nous cherchions à nous assurer que tout le contenu est facilement lisible, même par des non-experts, la nature du matériel peut signifier qu’il peut y avoir un niveau d’expertise et de compréhension technologique nécessaire pour comprendre certains chapitres et certaines mesures, qui peuvent ne pas répondre au <a href="https://www.w3.org/WAI/WCAG21/quickref/#reading-level" hreflang="en">niveau WCAG 2.1. des directives de niveau de lecture AAA</a>.
```
Comment (borisschapira) [inline src/templates/fr/2019/accessibility_statement.html:65]: Typo "Almanach"
+ Inclusivité

```suggestion
          Le but de l’Almanac Web est de fournir des informations détaillées sur l’état du Web. Bien que nous cherchions à nous assurer que tout le contenu est facilement lisible, même par des personnes non-expertes, la nature du matériel peut signifier qu’il peut y avoir un niveau d’expertise et de compréhension technologique nécessaire pour comprendre certains chapitres et certaines mesures, qui peuvent ne pas répondre au <a href="https://www.w3.org/WAI/WCAG21/quickref/#reading-level" hreflang="en">niveau WCAG 2.1. des directives de niveau de lecture AAA</a>.
```

File: src/templates/fr/2019/accessibility_statement.html [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+        </p>
+
+        <p>
+          Nous utilisons également Google Sheets pour certaines visualisations interactives, qui peuvent également ne pas être conformes aux normes d’accessibilité WCAG. Ceux-ci sont chargés via des <code>iframes</code>, nous fournissons des images pour toutes nos figures et affichons uniquement des visualisations interactives lorsque le logiciel client le prend en charge. Si vous utilisez un navigateur Web et que vous ne souhaitez pas afficher les visualisations interactives, les images peuvent être appliquées en ajoutant un paramètre <code>?print</code> pour tous les chapitres, <a href="./fr/2019/accessibility?print">consultez le chapitre Accessibilité en version imprimable</a> pour un exemple de ce type. Nous pouvons chercher à rendre cette fonctionnalité plus facile à activer si nécessaire, mais toutes les visualisations ont des descriptions textuelles détaillées, nous ne pensons donc pas que cela soit nécessaire. <a href="https://github.com/HTTPArchive/almanac.httparchive.org/issues" hreflang="en">Veuillez nous faire savoir</a> si nous nous trompons à ce sujet.
```
Comment (borisschapira) [inline src/templates/fr/2019/accessibility_statement.html:69]: Visualisations => celles-ci
+ reformulation dernière phrase

```suggestion
          Nous utilisons également Google Sheets pour certaines visualisations interactives, qui peuvent également ne pas être conformes aux normes d’accessibilité WCAG. Celles-ci sont chargées via des <code>iframes</code>, nous fournissons des images pour toutes nos illustrations et affichons uniquement des visualisations interactives lorsque le logiciel client le prend en charge. Si vous utilisez un navigateur Web et que vous ne souhaitez pas afficher les visualisations interactives, les images peuvent être appliquées en ajoutant un paramètre <code>?print</code> pour tous les chapitres, <a href="./fr/2019/accessibility?print">consultez le chapitre Accessibilité en version imprimable</a> pour un exemple de ce type. Nous pouvons chercher à rendre cette fonctionnalité plus facile à activer si nécessaire, mais toutes les visualisations ont des descriptions textuelles détaillées, nous ne pensons donc pas que cela soit nécessaire. Si vous pensez que c'est une erreur, <a href="https://github.com/HTTPArchive/almanac.httparchive.org/issues" hreflang="en">n'hésitez pas à nous le dire</a>.
```

File: src/templates/fr/2019/accessibility_statement.html [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+        </p>
+      </section>
+      
+      <section>
+        <h2 id="retour-information"><a href="#retour-information" class="anchor-link">Retour d’information</a></h2>
+
+        <p>
+          Nous apprécions les commentaires sur l’accessibilité de ce site Web. La meilleure façon de nous joindre serait de <a href="https://github.com/HTTPArchive/almanac.httparchive.org/issues" hreflang="en">ouvrir un ticket sur Github</a> ou de nous envoyer un courriel à <a href="mailto:team@httparchive.org">team@httparchive.org</a>. Nous serions également ravis de recevoir des pull-requests pour ceux qui souhaitent contribuer directement. Le Web Almanac est un projet communautaire, géré par des bénévoles, et nous ne pouvons donc pas garantir un délai pour répondre aux commentaires, mais nous y répondrons dès que possible.
```
Comment (borisschapira) [inline src/templates/fr/2019/accessibility_statement.html:120]: Inclusivité en changeant de voix (évite de faire de "nous" un masculin).

```suggestion
          Vos commentaires sur l’accessibilité de ce site Web nous intéressent. La meilleure façon de nous joindre serait <a href="https://github.com/HTTPArchive/almanac.httparchive.org/issues" hreflang="en">d’ouvrir un ticket sur Github</a> ou de nous envoyer un courriel à <a href="mailto:team@httparchive.org">team@httparchive.org</a>. Vous souhaitez contribuer directement ? N'hésitez pas à proposer vos <span lang="en">Pull Requests<span>, elles seront les bienvenues. Le Web Almanac est un projet communautaire, géré par des bénévoles, et nous ne pouvons donc pas garantir un délai pour répondre aux commentaires, mais nous y répondrons dès que possible.
```

File: src/templates/fr/2019/accessibility_statement.html (hunk 3 above)
Comment (nico3333fr) [reply src/templates/fr/2019/accessibility_statement.html]: Bien vu 👍

File: src/templates/fr/2019/accessibility_statement.html (hunk 7 above)
Comment (nico3333fr) [reply src/templates/fr/2019/accessibility_statement.html]: Excellente cette reformulation (et les autres aussi), merci !

**Comment by nico3333fr** [comment]
Thanks a lot @borisschapira your suggestions are top, as always 👍

File: src/templates/fr/2019/accessibility_statement.html [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+        </p>
+      </section>
+      
+      <section>
+        <h2 id="retour-information"><a href="#retour-information" class="anchor-link">Retour d’information</a></h2>
+
+        <p>
+          Vos commentaires sur l’accessibilité de ce site Web nous intéressent. La meilleure façon de nous joindre serait <a href="https://github.com/HTTPArchive/almanac.httparchive.org/issues" hreflang="en">d’ouvrir un ticket sur Github</a> ou de nous envoyer un courriel à <a href="mailto:team@httparchive.org">team@httparchive.org</a>. Vous souhaitez contribuer directement&nbsp;? N’hésitez pas à proposer vos <span lang="en">Pull Requests<span>, elles seront les bienvenues. Le Web Almanac est un projet communautaire, géré par des bénévoles, et nous ne pouvons donc pas garantir un délai pour répondre aux commentaires, mais nous y répondrons dès que possible.
```
Comment (borisschapira) [inline src/templates/fr/2019/accessibility_statement.html:120]: HTML typo, found by linter 

```suggestion
          Vos commentaires sur l’accessibilité de ce site Web nous intéressent. La meilleure façon de nous joindre serait <a href="https://github.com/HTTPArchive/almanac.httparchive.org/issues" hreflang="en">d’ouvrir un ticket sur Github</a> ou de nous envoyer un courriel à <a href="mailto:team@httparchive.org">team@httparchive.org</a>. Vous souhaitez contribuer directement&nbsp;? N’hésitez pas à proposer vos <span lang="en">Pull Requests</span>, elles seront les bienvenues. Le Web Almanac est un projet communautaire, géré par des bénévoles, et nous ne pouvons donc pas garantir un délai pour répondre aux commentaires, mais nous y répondrons dès que possible.
```

Comment (nico3333fr) [reply src/templates/fr/2019/accessibility_statement.html]: Ah, I was searching where the linter found it, you were faster :D

**Comment by bazzadp** [comment]
Thanks both - great work!

**Pull request merged**